Hidden vs. Readonly Properties (#242)
replicatedhq/kots.io: content/reference/v1beta1/config.md

          value: "{{repl RandomString 40}}"
```

> When used in conjunction with [RandomString](/reference/template-functions/static-context/#randomstring)

Reviewer: its my experience that readonly affect this. does hidden?

Author: It doesn't but it generally is used with `hidden` If its not a hidden field they can edit it.

          readonly: true
```

> 

Reviewer: My understanding is it’s more than just random string

Author: @emosbaugh Oh! So does that mean _any_ function that generate "things" (RandomString being just one of them) will regenerate based on `hidden` vs `readonly`?

Reviewer 2: I think that's right @aj-jester but I'll let @emosbaugh confirm

Author: Updated the doc to be more generic.

> When used in conjunction with [RandomString](/reference/template-functions/static-context/#randomstring)

Reviewer: its more than just random string. i think we should generalize this

Author: Updated to make it more generic.

- If set to `true`, the `value` is **ephemeral** between Config changes. It **cannot** be modified because it is greyed out in HTML.
- If not set or set to `false`, the `value` is **persistent** between Config changes. It **can** be modified because its not greyed out in HTML.

### `affix`
Items can be affixed left or right. These items will appear in the admin console on the same line.
```yaml
    affix: left
```

### `help_text`
This is similar to `description` but for `items`. This property can show a helpful message below `title`. [Markdown](https://guides.github.com/features/mastering-markdown/) syntax is supported.
```yaml
    - name: toggles
      items:
      - name: http_enabled
        title: HTTP Enabled

>         help_text: When enabled we will listen to http

Reviewer 2: kind of unrelated but I'm not totally a fan of this kind of language in a config screen, and I don't think we should encourage vendors to do this in their demo apps. Who is `we` in this case?

how about

```suggestion
        help_text: Check to enable the HTTP listener
```
